feature-libs/product/configurators/common/common-configurator.module.ts

```diff
@@ -10,4 +12,13 @@ import { CommonConfiguratorCoreModule } from './core/common-configurator-core.mo
     ConfigurationMessageLoaderModule,
   ],
 })
-export class CommonConfiguratorModule {}
+export class CommonConfiguratorModule {
+  static withConfig(
```

Comment:
As noted in another comment https://github.com/SAP/spartacus/pull/8918#discussion_r490320458, it would be good to drop this function